exasol/toolbox/nox/_release.py

```diff
@@ -29,13 +33,17 @@
 def _create_parser() -> argparse.ArgumentParser:
     parser = argparse.ArgumentParser(
         prog="nox -s release:prepare",
-        usage="nox -s release:prepare -- [-h] version",
+        usage="nox -s release:prepare -- [-h] [-t | --type {major,minor,patch}]",
         formatter_class=argparse.ArgumentDefaultsHelpFormatter,
     )
     parser.add_argument(
-        "version",
-        type=cli.version,
-        help=("A version string of the following format:" '"NUMBER.NUMBER.NUMBER"'),
+        "-t",
+        "--type",
+        type=ReleaseTypes,
+        help="specifies which type of upgrade is to be performed",
+        required=True,
+        choices=[rt.value for rt in list(ReleaseTypes)],
```

Comment:
With Python 3.10, the `choices` & `type` are leading to conflicts.
`nox -s release:prepare: error: argument -t/--type: invalid choice: <ReleaseTypes.Major: 'major'> (choose from 'major', 'minor', 'patch')`

I'd recommend removing the choices one, as the type already propagates the help text. 

Please verify that the change works for you locally by running the `release:prepare`, after commenting out the GitHub PR creation line. Before pushing any changes, make sure to drop the commit where the release was prepared.
```suggestion
```